src/Country.php

```diff
@@ -1,85 +1,37 @@
 <?php
 namespace MoneyPHP\IsoCurrencies;
 
-/**
- * Class Country
- * @package MoneyPHP\IsoCurrencies
- */
 final class Country
 {
-    /**
-     * @var string
-     */
-    private $alphabeticCode;
-    /**
-     * @var string
-     */
-    private $currency;
-    /**
-     * @var string
-     */
-    private $entity;
-    /**
-     * @var int
-     */
-    private $minorUnit;
-    /**
-     * @var int
-     */
-    private $numericCode;
-
-    /**
-     * @param string $entity
-     * @param string $currency
-     * @param string $alphabeticCode
-     * @param int $numericCode
-     * @param int $minorUnit
-     */
-    public function __construct($entity, $currency, $alphabeticCode, $numericCode, $minorUnit)
+    public function __construct(private readonly string $entity,
+                                private readonly string $currency,
+                                private readonly string $alphabeticCode,
+                                private readonly int    $numericCode,
+                                private readonly int    $minorUnit)
```

Comment:
Add new line, identation

Reply:
i changed identation betwen type and variable name. I am not sure if you that is what you mean.

I am not sure where exact you want me to add the line